### PR TITLE
removing voids from arity

### PR DESCRIPTION
**Pull request by CrossEye**
Like #698.  Works in `grunt jshint`.  Does this fail in `make`?

### CODE REVIEW

**Comment by megawac** [comment]
:+1: but what happens when you minify it?

**Comment by CrossEye** [comment]
@megawac:  Not sure what the question is.  So the comment is stripped out and it wont lint properly.  But a minified file shouldn't be passed to lint anyway, right?  Is there some other concern?

**Comment by megawac** [comment]
Will the parameter be removed in the minifed build (i.e. the arity be set incorrectly)

File: dist/ramda.js [hunk 1]
```diff
@@ -515,7 +515,9 @@
      *      // All arguments are passed through to the wrapped function
      *      takesOneArg(1, 2); //=> [1, 2]
      */
+    /*jshint unused:false */
     var arity = function (n, fn) {
+        /*jshint unused:false */
```
Comment (megawac) [inline dist/ramda.js]: why is it duplicated

**Comment by CrossEye** [comment]
Ahh, I didn't know minifiers might do that.  I'll check.

**Comment by CrossEye** [comment]
Ahh, looks like the minification is a problem:

> WARN: Dropping unused function argument a0 [-:527,29]

I'm going to close this one for now.  Maybe at some point we can also configure the minifier, but that's getting a fair bit trickier.

**Pull request closed without merging**